### PR TITLE
Fix trailing _ on "panos_aggregate_interface"

### PR DESCRIPTION
**Pull request by IAmStoxe**
Was failing upon an import with an aggregate interface defined (error below).

I found there was a trailing underscore specified within panos_aggregate_interface within `providers/panos/firewall_networking.go`

```
2021/08/30 23:31:33 Refreshing state... panos_aggregate_interface_.tfer--ae1
panic: unknown resource type panos_aggregate_interface_

goroutine 157 [running]:
github.com/hashicorp/terraform/plugin.(*GRPCProvider).getResourceSchema(0xc0008b63f0, 0xa027083, 0x1a, 0x0, 0x0)
        /go/pkg/mod/github.com/hashicorp/terraform@v0.12.31/plugin/grpc_provider.go:92 +0x12a
github.com/hashicorp/terraform/plugin.(*GRPCProvider).ReadResource(0xc0008b63f0, 0xa027083, 0x1a, 0xadea7f8, 0xc002041ed0, 0x8ecff80, 0xc002109a40, 0xfd11cb0, 0x0, 0x0, ...)
        /go/pkg/mod/github.com/hashicorp/terraform@v0.12.31/plugin/grpc_provider.go:321 +0xa5
github.com/GoogleCloudPlatform/terraformer/terraformutils/providerwrapper.(*ProviderWrapper).Refresh(0xc0008b6000, 0xc000f3fbd0, 0xc000f3fb80, 0x2, 0xc00205c400, 0x39)
        /tmp/terraformer/terraformutils/providerwrapper/provider.go:171 +0x305
github.com/GoogleCloudPlatform/terraformer/terraformutils.(*Resource).Refresh(0xc000b593b0, 0xc0008b6000)
        /tmp/terraformer/terraformutils/resource.go:130 +0x4f
github.com/GoogleCloudPlatform/terraformer/terraformutils.RefreshResourceWorker(0xc000a46900, 0xc00092c150, 0xc0008b6000)
        /tmp/terraformer/terraformutils/utils.go:153 +0xcd
created by github.com/GoogleCloudPlatform/terraformer/terraformutils.RefreshResources
        /tmp/terraformer/terraformutils/utils.go:80 +0x145
```

### CODE REVIEW

**Comment by IAmStoxe** [comment]
@googlebot I signed it!

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: "tagged-release"
+on:
```
Comment (sergeylanzman) [inline .github/workflows/go.yml:2]: ?

**Comment by IAmStoxe** [comment]
That never should've made it in. When I edited my github actions it updated the pull automatically and ammended the commit..... I have no idea why

File: .github/workflows/go.yml (hunk 1 above)
Comment (sergeylanzman) [reply .github/workflows/go.yml:2]: @IAmStoxe why it's copy file?

**Pull request merged**